### ISSUE TITLE
getting error (IMPORT FAILED)

### ISSUE THREAD

**Issue by chnisar515** [gokayfem/ComfyUI_VLM_nodes#116]
Hello Experts,
i fresh installed VLM today but its showing me (IMPORT FAILED) error. i have installed it via Manager. Any solution for this please.

**Comment by vatsal-vb** [comment]
I am getting import error and conflict with Comfyui-yolo, even though don't have it comfyui-yolo. 
<img width="1788" alt="Screenshot 2024-10-24 at 05 30 38" src="https://github.com/user-attachments/assets/25c95629-da11-4526-a2a7-01684a0fb692">

**Comment by MG-Soliman** [comment]
Same issue here, caused by this error : (latest packages and comfy 0.2.4)

Installing llama-cpp-python...
Looking in indexes: https://jllllll.github.io/llama-cpp-python-cuBLAS-wheels/AVX2/cu124
ERROR: Could not find a version that satisfies the requirement llama-cpp-python (from versions: none)
ERROR: No matching distribution found for llama-cpp-python

**Comment by DominikBoettger** [comment]
> Same issue here, caused by this error : (latest packages and comfy 0.2.4)
> 
> Installing llama-cpp-python... Looking in indexes: https://jllllll.github.io/llama-cpp-python-cuBLAS-wheels/AVX2/cu124 ERROR: Could not find a version that satisfies the requirement llama-cpp-python (from versions: none) ERROR: No matching distribution found for llama-cpp-python

Got the exact same error. Did anybody find a solution?

**Comment by dkqjhoifh** [comment]
me too

**Comment by sanebg** [comment]
what a piece of shit node. The dev should do something else instead of wasting peoples time. It's free and I still want my money back/

**Comment by iEddie-cmd** [comment]
Same. Pls fix anyone

**Comment by EnragedAntelope** [comment]
That's pretty harsh. These are good nodes. Something changed with that source repo because this is a recent issue. I was able to git clone https://github.com/gokayfem/ComfyUI_VLM_nodes into my Comfy Custom Nodes folder, and manually update llama-cpp-python from https://github.com/abetlen/llama-cpp-python to get it working. 
I am talking to the dev to try to figure out better logic for handling llama-cpp-python, which is only necessary really for older llava models at this point.

**Comment by iEddie-cmd** [comment]
I cloned it too and run CMAKE_ARGS="-DGGML_CUDA=on" pip install --upgrade --force-reinstall llama-cpp-python but it did not fix the issue for me.

**Comment by EnragedAntelope** [comment]
> I cloned it too and run CMAKE_ARGS="-DGGML_CUDA=on" pip install --upgrade --force-reinstall llama-cpp-python but it did not fix the issue for me.

did you get an error on install of llama-cpp-python? or did you get an error in the comfy log? if so please share any errors so we can try to figure out how to address

**Comment by gokayfem** [comment]
It should be fixed now. Thanks for your patience.

**Comment by MG-Soliman** [comment]
still as broken , at this point i guess i should accept this is never going to work

**Comment by gokayfem** [comment]
can you send me the error on your cmd screen, some kind of logs

**Comment by DominikBoettger** [comment]
It's working perfectly for me. Thank you gokayfem, much appreciated!

**Comment by highfiiv** [comment]
Hmm, @gokayfem looks like I have the latest version but do get:
```
[!] ERROR: Could not find a version that satisfies the requirement decord (from versions: none)
[!] ERROR: No matching distribution found for decord
install script failed: https://github.com/gokayfem/ComfyUI_VLM_nodes
```
On Macos

**Comment by gokayfem** [comment]
on macos please pip install eva-decord

**Comment by highfiiv** [comment]
Hi @gokayfem that did not work, are there any other dependencies I should be aware of?
My CUI functions well.

**Comment by MG-Soliman** [comment]
things are working again after a clean install on the latest comfyui v1 closed beta  0.2.9 - Build 241106m7net450s-x64 , thank you <3

**Comment by OhShush** [comment]
Hello, I seem to still have this issue. Errors present on restarting ComfyUI:

```
0.6 seconds (IMPORT FAILED): /Users/My_Username/ComfyUI/custom_nodes/ComfyUI_VLM_nodes

ERROR: Could not find a version that satisfies the requirement decord (from versions: none)
ERROR: No matching distribution found for decord

```

running pip install eva-decord yields:

```
(base) My_Username@Mac ComfyUI % pip install eva-decord
Requirement already satisfied: eva-decord in /opt/anaconda3/lib/python3.11/site-packages (0.6.1)
Requirement already satisfied: numpy>=1.14.0 in /opt/anaconda3/lib/python3.11/site-packages (from eva-decord) (1.26.4)
```

Completely reinstalled ComfyUI and ComfyUI manager from scratch today. Any suggestions?
